torch/_subclasses/functional_tensor.py

```diff
@@ -153,7 +154,7 @@ def __repr__(self):
         return f"FunctionalTensor({repr(self.elem)})"
 
     @staticmethod
-    def to_functional(x):
+    def to_functional(x, is_pre_dispatch=False):
```

Comment:
Hmm - I think we mentioned a few days ago that it would be nice to have a util that says "please grab the current  (global) FunctionalTensorMode, which is guarnateed to be in either the PreDispatch or Python mode stack, but not both".

If we have that, then we could just grab that mode and enable it here, instead of needing to plumb a kwarg and create a fresh mode below.

Reply:
Yep i did that in the later PR. Didn't want to make this PR too complicated

Reply:
sounds good! (as long as you clean this up when the new util is added)